runtimes/alibi-detect/mlserver_alibi_detect/runtime.py

```diff
@@ -0,0 +1,102 @@
+from mlserver import types
+from mlserver.settings import ModelSettings
+from mlserver.model import MLModel
+from mlserver.errors import InferenceError
+from mlserver.codecs import NumpyCodec
+from fastapi import Request, Response
+from mlserver.handlers import custom_handler
+from .protocols.util import (
+    NumpyEncoder,
+    get_request_handler,
+    Protocol,
+)
+import numpy as np
+import json
+from typing import Optional, Any
+
+# from pydantic import BaseSettings
+
+# ENV_PREFIX_ALIBI_DETECT_SETTINGS = "MLSERVER_MODEL_ALIBI_DETECT_"
+
+
+class AlibiDetectSettings(dict):
+    """
+    Parameters that apply only to alibi detect models
+    """
+
+    def __init__(self, dictionary):
```

Comment:
Isn't this one the default behaviour for Pydantic models?

Reply:
I was getting some issues with the `extra` dict being used in pydantic `expected str, bytes or os.PathLike object, not dict`